octavia-cli/setup.py

```diff
@@ -15,7 +15,7 @@
 
 setup(
     name="octavia-cli",
-    version="0.1.0",
+    version="0.35.65",
```

Comment:
> Is this supposed to be kept in sync with the Airbyte version?

Yes! Octavia is released along Airbyte and keeps the same versioning. Python package versioning convention does not support the `-alpha` prefix. This is why I cut the suffix. I created a specific bump version config for this file in `.bumpversion.cfg` in this PR.
I did this in this PR because it was the cleanest way of retrieving the current package version from the Python app.